### ISSUE TITLE
Missing recipe categories

### ISSUE THREAD

**Issue by nea89o** [nea89o/Firmament#7]
- [ ] Mob drops
- [ ] Kat upgrades
- [ ] Essence upgrades
- [ ] NPC Shops
- [ ] Villager trades (stranded)
- [ ] Wiki data as an extra tab.

**Comment by Apceniy** [comment]
What about brewing recipes?

**Comment by nea89o** [comment]
Yes, but that's not a thing in the NEU repo right now. In principle i could support it (and will in the future), but this issue is more about parity with the repo. Also cc @walkerselby who might do this one at a repo level for neu (or might not, he is busy with lots of more important work)
